docs/glossary.rst

```diff
@@ -153,3 +155,16 @@
       A library for operating with URL objects.
 
       https://pypi.python.org/pypi/yarl
+
+
+Enviornment Variables
```

Comment:
```suggestion
Environment Variables
```

Reply:
Whoops, thanks for catching this, @Dreamsorcerer